apps/site/src/utils/index.ts

```diff
@@ -4,6 +4,15 @@ import { useAtom } from 'jotai';
 import { useCallback, useEffect, useMemo } from 'react';
 import { addressAtom, userAtom } from '../atoms';
 
+export const config = () =>
+  fcl.config({
+    'accessNode.api': 'https://rest-mainnet.onflow.org',
+    'app.detail.title': 'My Flow NFTs',
+    'discovery.wallet': 'https://flow-wallet.blocto.app/authn',
+    '0xProfile': '0x97bafa4e0b48eef',
```

Comment:
should these addresses be hardcoded?

Reply:
fcl.config links in both `main` and `find-auth-example` (https://github.com/lightbeardao/find-auth-example/blob/473348b501fbaa7b7becfc618d91088efc93086c/lib/config.js) are hardcoded. Will check if we can move these to .env instead